azerothcore/azerothcore-wotlk
# Add GitRevision

**Changes proposed:**
- Add GitRevision as in TrinityCore
- Delete SystemConfig.h

#### When you change the revision, some files (described below) will not be compiled again
* src/common/Debugging/WheatyExceptionReport.cpp
* src/server/authserver/Main.cpp
* src/server/game/Handlers/CharacterHandler.cpp
* src/server/game/World/World.cpp
* src/server/scripts/Commands/cs_server.cpp
* src/server/worldserver/Master.cpp

**Target branch(es):** master.

**Issues addressed:** No issue

**Tests performed:** Tested build (Windows 10 and Linux 14.04)

**How to test the changes:**
- Apply commit (https://github.com/Winfidonarleyan/azerothcore-wotlk/commit/a851f4496842cf2a2c6263a400641f19cc151308)
- Rebuild core
- Add new test commit
- Rebuild core
- You can see how only 1 file is compiled (GitRevision.cpp)

**Known issues and TODO list:** No
